BossMod/Modules/Endwalker/HuntS/Armstrong.cs

```diff
@@ -22,50 +23,86 @@ public enum AID : uint
         SoporificGas = 27478, // Boss->self, 6.0s cast, range 12 circle
     };
 
-    class MagitekCompressor : Components.GenericAOEs
+    class MagitekCompressor : Components.GenericRotatingAOE
```

Comment:
This one seems to be overcomplicated - surely you can just invert the sequence's increment after 5th cast?

Reply:
i am not sure if I understand what you mean, but the activation delay is different (3.6s instead of 2.1s) for the cast where it is inverting other than that already is inverting the increment after the 5th cast